contrib/README.md

```diff
@@ -0,0 +1,44 @@
+# ovn kubernetes KIND Setup
+
+KIND deployment of OVN kubernetes is a fast and easy means to quickly install and test kubernetes with OVN kubernetes CNI
+
+### Prerequisites
+
+- 20 GB of free space in root file system
+- Docker run time
+- Kind  ( https://kubernetes.io/docs/setup/learning-environment/kind/ )
```

Comment:
I tend to use all caps for KIND. Not sure if it correct or not.

Reply:
Thanks @Billy99 . I'll add this to the document you suggested. I would prefer to keep the readme file that just points to the main document. That way it'll be made more visible and user friendly.

Reply:
@Billy99  i updated the docs as discussed above.